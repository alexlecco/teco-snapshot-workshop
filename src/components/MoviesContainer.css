h4 {
  font-size: 15px;
  margin: 10px;
}
