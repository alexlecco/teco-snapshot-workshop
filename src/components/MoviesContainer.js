import React from 'react';
import './MoviesContainer.css';

const MoviesContainer = ({ movies = [] }) => {
  if (movies.length <= 0) {
    return <>No se encontraron pel√≠culas</>
  } else {
    return <>
      <ol>
        {movies.map((item, index) => {
          return <li key={index}>{item}</li>
        })}
      </ol>
    </>
  }
};

export default MoviesContainer;
