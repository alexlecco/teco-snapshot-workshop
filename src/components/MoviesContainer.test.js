import React from 'react';
import renderer from 'react-test-renderer';

import MoviesContainer from './MoviesContainer';

it ('renderizar mensaje cuando no hay peliculas', () => {
  const tree = renderer.create(<MoviesContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});

it ('renderizar cuando hay solo 1 pelÃ­cula', () => {
  const tree = renderer.create(<MoviesContainer movies={['Shrek']} />).toJSON();
  expect(tree).toMatchSnapshot();
});
