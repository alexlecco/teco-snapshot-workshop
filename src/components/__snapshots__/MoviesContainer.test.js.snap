// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderizar cuando hay solo 1 película 1`] = `
<h4>
  Shrek
</h4>
`;

exports[`renderizar mensaje cuando no hay peliculas 1`] = `"No se encontraron películas"`;
